### ISSUE TITLE
Allow empty EIDs field

### ISSUE THREAD

**Issue by bretg** [prebid/prebid-server#2712]
https://github.com/prebid/Prebid.js/pull/9841 reports that PBS (at least Java) is choking on an empty user.eids:

```
request.user.eids must contain at least one element or be undefined
```

I don't recall why we would be validating this field -- we should discuss this and other empty-array locations.

**Comment by bretg** [comment]
PBS-Go also rejects an empty EIDs array. Besides that validation here are the others:

- source must be defined and unique
- must have at least one uids entry
- each uids entry must have an id

We're ok relaxing this rule. If a bid adapter has trouble with an empty user.eids, we will reach out to the bidder and work with them.

**Comment by SyntaxNode** [comment]
Implemented in PBS-Go 0.253
Released with PBS-Java 1.119
